gramineproject/gramine
# Drop hacks and workarounds for Ubuntu 18.04

## Description of the changes 

For now only the simpler ones, more complicated ones will go into separate PRs.

## How to test this PR? 

Check if it still works on all supported platforms.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/1373)
